src/core/client_channel/retry_throttle.cc

```diff
@@ -18,16 +18,32 @@
 
 #include "src/core/client_channel/retry_throttle.h"
 
-#include <grpc/support/atm.h>
 #include <grpc/support/port_platform.h>
 
+#include <atomic>
+#include <limits>
 #include <map>
 #include <string>
 #include <utility>
 
+#include "src/core/util/useful.h"
+
 namespace grpc_core {
 namespace internal {
 
+namespace {
+template <typename T>
+T ClampedAdd(std::atomic<T>& value, T delta, T min, T max) {
```

Comment:
Does this need to be a template?  Seems like we're only ever calling it with one type.

Reply:
I'd *like* to have more sensible types for it in the future, but it needs more work (and I don't want to have to keep changing this)